plugins/vendor/zoom/ZoomDropFolderPlugin/batch/processors/zoomRecordingProcessor.php

```diff
@@ -354,11 +389,18 @@ protected function getValidatedHosts($recordingId, $userToExclude)
 		$userToExclude = strtolower($userToExclude);
 		$zoomCoHosts = $this->getCoHostsEmails($recordingId);
 		$zoomAlternativeHosts = $this->getAlternativeHostsEmails($recordingId);

```

Comment:
if we have now function for getValidatedAlternativeHosts why do we add them here under the hosts function?
i would divide it completely and call the function name getValidatedHosts